.github/workflows/CI.yml

```diff
@@ -47,6 +47,22 @@ jobs:
         with:
           NO_COVERAGE: ${{ matrix.NO_COVERAGE }}
 
+  test-cygwin:
+    name: "cygwin64 - GAP master - HPCGAP no"
+    if: ${{ !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository) }}
+    runs-on: windows-latest
+    steps:
+      - name: "Set git to use UNIX-style line endings"
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
```

Comment:
Is this really needed? What happens if we drop it?
```suggestion
```

Reply:
It fails once it tries to set up GAP:
```
D:\a\io\io>C:\cygwin64\bin\bash -l -c "export GAP_PKGS_TO_BUILD=\"\" && pkg-ci-scripts/build_gap.sh" 
pkg-ci-scripts/build_gap.sh: line 13: set: -
: invalid option
set: usage: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
pkg-ci-scripts/build_gap.sh: line 14: $'\r': command not found
pkg-ci-scripts/build_gap.sh: line 16: $'\r': command not found
Cloning into '/home/runneradmin/gap

[...omitted...]

pkg-ci-scripts/build_gap.sh: line 20: $'\r': command not found
pkg-ci-scripts/build_gap.sh: line 47: syntax error near unexpected token `$'do\r''
pkg-ci-scripts/build_gap.sh: line 47: `for pkg in ${GAP_PKGS_TO_CLONE}; do
'
Error: Process completed with exit code 2.
```